gradle.properties

```diff
@@ -12,4 +12,7 @@ android.useAndroidX=true
 # Automatically convert third-party libraries to use AndroidX
 android.enableJetifier=true
 # Kotlin code style for this project: "official" or "obsolete":
-kotlin.code.style=official
\ No newline at end of file
+kotlin.code.style=official
+android.defaults.buildfeatures.buildconfig=true
+android.nonTransitiveRClass=false
+android.nonFinalResIds=false
```

Comment:
Why are these necessary?

`nonTransitiveRClass` is in the default gradle.properties file generated by the Android Studio new project wizard, but with a value of `true`.

The other two are not in the default gradle.properties file at all.

Reply:
I have used Android Studio migration assistant, this two lines had been added automatically. I don't yet their impact on the project, it will require an extra research to figure out. 

If tests passed, in my opinion we can merge it into the master.

Reply:
I've reverted to the gradle.properties from the new project wizard, because that's more representative of what most users will have.